doc/user/routemap.rst

```diff
@@ -151,10 +151,15 @@ Route Map Match Command
 
    Matches the specified `prefix-len`. This is a Zebra specific command.
 
-.. index:: match ip next-hop IPV4_ADDR
```

Comment:
I see we have this command. With your changes both are valid: `match ip next-hop address <IP>` and `match ip next-hop <IP>`.

Reply:
"match ip next-hop <IP>." does not exist

Reply:
`match ip next-hop ` falls on acl name.

Reply:
ah, yes.